src/tomczyk-kamil/Fsd.Kamil.Cs/Fsd.Kamil.Cs.Ex2/Fsd.Kamil.Cs.Ex2/Fsd.Kamil.Cs.Ex2/Program.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Fsd.Kamil.Cs.Ex2.Animals;
+
+namespace Fsd.Kamil.Cs.Ex2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            FarmMaker newFactory = new FarmMaker();
+            newFactory.NewAnimal();          
```

Comment:
nazwa NewAnimal() sugeruje mi że utworzy się tylko 1 zwierze i nic więcej się nie stanie. To chyba nie jest do końca prawda? Może inna nazwa